File: java/src/Ui/StockReportCommand.java
package Ui;

import Warehouse.Warehouse;

class StockReportCommand implements UserCommand {

	private Warehouse warehouse;

	StockReportCommand(Warehouse warehouse) {
		this.warehouse = warehouse;
	}

	public void run(UserInput cmd) {
		warehouse.stockReport(System.out);
	}

}

File: csharp/WarehouseActions/StockReportCommand.cs
namespace WarehouseActions {

	using Ui;
	using Warehouse;

	class StockReportCommand : UserCommand {

		private Warehouse warehouse;

		internal StockReportCommand(Warehouse warehouse) {
			this.warehouse = warehouse;
		}

		public void Run(UserInput cmd) {
			warehouse.StockReport();
		}

	}
}